index.js

```diff
@@ -221,4 +174,8 @@ module.exports = function (env) {
   function withTransaction(statements) {
     return using(getTransactionWithEnv(), statements)
   }
+
+  function on(event, fn) { pool.on(event, fn) }
```

Comment:
```suggestion
  function on(event, fn) { return pool.on(event, fn) }
```

It would be nice if one could chain multiple listeners. This also matches the usual EventEmitter semantics.